KthSmallestElementInASortedMatrix.py

```diff
@@ -0,0 +1,58 @@
+class Solution(object):
+    def kthSmallest(self, matrix, k):
+        """
+        :type matrix: List[List[int]]
+        :type k: int
+        :rtype: int
+        """
+        if not matrix:
+            return 0
+        m = len(matrix)
+        n = len(matrix[0])
+        heap = []
```

Comment:
Apparently, this is not OOP.
Should define a class Heap and define its methods.

Reply:
Comments has been addressed.